src/tools/python/loops.md

```diff
@@ -66,6 +66,16 @@ for cool_var in range(7):
 	print(f"This works! {cool_var}")
 ```
 
+sometimes, we may need to keep track of the index of some iterable. We can take advantage of `len()` from before to help us solve these problems:
```

Comment:
erm akshually the best way of doing that is with the enumerate generator

granted I'd rather personally implement that in the iterator and listcomp section, you can disregard this comment